nrf_security/src/mbedcrypto_glue/cc310/ecdsa_cc310.c

```diff
@@ -32,7 +32,7 @@ static int mbedtls_ecdsa_check(mbedtls_ecp_group *grp, mbedtls_ecp_group_id gid,
         case MBEDTLS_ECP_DP_SECP192K1:
         case MBEDTLS_ECP_DP_SECP224K1:
         case MBEDTLS_ECP_DP_SECP256K1:
-            return 1;
+            return 3;
```

Comment:
1 to 3  ?

Reply:
Yes. This was corrected from initial value (which should have been 2), but modified to 3 to give room for Oberon (which is priority 2)